### ISSUE TITLE
[DOCS] triggers given in the user management app tutorial (react) doesn't work

### ISSUE THREAD

**Issue by Ruchita1010** [supabase/supabase#21902]
# Improve documentation

## Link 

[user management app with react](https://supabase.com/docs/guides/getting-started/tutorials/with-react)

## Describe the problem

I added the triggers to remove the old avatars of a user as per [this](https://supabase.com/docs/guides/getting-started/tutorials/with-react#storage-management) tutorial in the docs but they don't seem to work. I've enabled the http extension and also made sure that the triggers are added to the database. 

![trigger in the dashboard](https://github.com/supabase/supabase/assets/70577616/249a94e7-ea0f-44aa-a8bd-59beba4ae697)

![multiple avatars in storage](https://github.com/supabase/supabase/assets/70577616/69aef132-7ea5-492b-a263-415b95d61be6)

**Comment by encima** [comment]
Thanks for reporting.

Can you check the logs from Postgres and confirm the trigger is firing, as well as the outcome of the function. Can you also confirm you are using the service key and not the anon key?

**Comment by Ruchita1010** [comment]
Yeah, so the trigger is firing but it's resulting in a 404 error. And I checked the function, I'm using the service key.

![postgrest-logs](https://github.com/supabase/supabase/assets/70577616/2ec1fa36-fefa-45fc-af4d-e745e6e51591)

**Comment by encima** [comment]
Can you confirm that the object does exist?

**Comment by Hallidayo** [comment]
Hi all - I've just ran the guide on my end and all seems okay with triggers firing correctly, images deleting etc...

@Ruchita1010 - When you do the upload do you get any errors on the frontend like an alert or errors in the network tab?
